cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -349,6 +362,7 @@ private void persistAggregation(NGSIGenericAggregator aggregator, String service
                 aggregation += "," + jsonObject;
             }
         }
+        entityId=aggregator.getEntityId();
```

Comment:
The goal for this task is to use the entity id as orgName, pkgName and resName. 

Perhaps It isn't necessary to store again the entityId on the sink, because it's already stored into the corresponding objects inside the generic aggregator.

do you agree @VicenteLT?

Reply:
fixed in dd61bbdef9e122d6e143d15f9d8e497f9697fdce